sdk/iot/README.md

```diff
@@ -1,14 +1,32 @@
 # Azure IoT Clients
 
-Azure SDK for Embedded C official IoT Hub client library.
+Azure SDK for Embedded C official IoT client library.
 
 ## Getting Started
 
-- **Docs**: If you're new to the Azure SDK for Embedded C take a look at the [State Machine diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities. 
+The IoT Client library is created to facilitate connectivity to the Azure IoT Hub and Azure IoT Provisioning Service alongside an MQTT and TLS stack of the user's choice. This means that this SDK is **NOT** a platform but instead is a true library.

```

Comment:
We have many clients: hub, provisioning. We'll likely add more in the future.
Maybe we should be less specific here and just say it's to facilitate devices connecting to Azure IoT services?

Reply:
We should also be careful about protocols: likely we'll use protocols other than MQTT - I'd hate to be the one changing everywhere when that happens.

Reply:
Yes to the first point, the second: ehhhhhh maybe but then we can do a find-all MQTT. I think for now I'd like to be that specific since right now MQTT is all we support. No use saying "a transport and tls of your choice" and leave them guessing which transports are allowed.